### DISCUSSION TITLE
editbuttontrue без иконки. Редактирование по клику на всем поле

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#167] [Есть идея, а что если...]
Пожелание. Вдруг получится сделать возможность редактирования полей в хедере и футере без иконки редактирования. Просто по клику на поле. Т. е. убрать иконку при editbutton = true распостранить ее действие на все поле ячейки  

<img width="585" alt="Снимок экрана 2022-07-18 в 11 38 54" src="https://user-images.githubusercontent.com/91674114/179474626-18d0fe40-5279-4544-943d-d5e8dcb3a1f2.png">

Люди не любят двойных кликов. В результате в интерфейсе образуется множество кнопок редактирования поля

**Comment by ttm-alexeypolunin** [comment]
Угу, запишем в список улучшений.
